<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgDeformOptions</class>
 <widget class="QWidget" name="WdgDeformOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>415</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item column="0" row="0">
    <widget class="QLabel" name="deformLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Deform Mode</string>
     </property>
    </widget>
   </item>
   <item column="0" row="1">
    <layout class="QGridLayout" name="modesGrid">
     <item column="0" row="0">
      <widget class="QRadioButton" name="growBtn">
       <property name="text">
        <string comment="Grow as in deform brush engine">Grow</string>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="QRadioButton" name="shrinkBtn">
       <property name="text">
        <string comment="Shrink as in deform brush engine">Shrink</string>
       </property>
      </widget>
     </item>
     <item column="0" row="1">
      <widget class="QRadioButton" name="swirlCCWBtn">
       <property name="text">
        <string>Swirl CCW</string>
       </property>
      </widget>
     </item>
     <item column="1" row="1">
      <widget class="QRadioButton" name="swirlCWBtn">
       <property name="text">
        <string>Swirl CW</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item column="0" row="2">
      <widget class="QRadioButton" name="moveBtn">
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
     </item>
     <item column="1" row="2">
      <widget class="QRadioButton" name="colorBtn">
       <property name="text">
        <string comment="Rearrange the pixels positions under the cursor">Color deformation</string>
       </property>
      </widget>
     </item>
     <item column="0" row="3">
      <widget class="QRadioButton" name="lensOutBtn">
       <property name="text">
        <string>Lens zoom out</string>
       </property>
      </widget>
     </item>
     <item column="1" row="3">
      <widget class="QRadioButton" name="lensBtn">
       <property name="text">
        <string>Lens zoom in</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="deformAmountLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="deformAmountLbl">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Deform amount:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KisDoubleSliderSpinBox" name="deformAmount"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="middleSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>105</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item column="0" row="3">
    <layout class="QVBoxLayout" name="qualityLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="interpolationChBox">
       <property name="text">
        <string>Bilinear interpolation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useCounter">
       <property name="text">
        <string>Use counter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useOldData">
       <property name="text">
        <string>Use undeformed image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="4">
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
